app/controllers/application_controller.rb

```diff
@@ -54,6 +54,7 @@ def with_current_user
   def current_person
     current_user.try(:person)
   end
+  helper_method :current_person
```

Comment:
this should be moved to the top of the file, next to `helper :all`

Reply:
I grouped all helper methods together.